src/JBMultiTerminal.sol

```diff
@@ -85,6 +85,9 @@ contract JBMultiTerminal is JBPermissioned, Ownable, ERC2771Context, IJBMultiTer
     /// deployment process.
     uint256 private constant _FEE_BENEFICIARY_PROJECT_ID = 1;
 
+    /// @notice 28 days, the number of seconds a fee can be held for.
```

Comment:
Could make it 32 days, so its always longer than a month from the date when the fee was registered